cli/tests/test_next_prev_commands.rs

```diff
@@ -567,6 +567,124 @@ fn test_next_editing() {
     "###);
 }
 
+#[test]
+fn test_prev_conflict() {
+    // Make the third commit our new parent.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let file_path = repo_path.join("content.txt");
+    std::fs::write(&file_path, "second").unwrap();
```

Comment:
nit: I generally count the root commit as the zeroth commit, so I find it a bit confusing to call the first commit after that "second". Could you update all these strings to be "first", "second", etc. instead of "second", "third", etc.?